nautobot/core/utils/requests.py

```diff
@@ -47,12 +47,11 @@ def nautobot_serialize(self):
         data = copy.deepcopy(self.__dict__)
         # We don't want to try to pickle/unpickle or serialize/deserialize the actual User object,
         # but make sure we do store its PK so that we can look it up on-demand after being deserialized.
+        user = data.pop("user")
+        data.pop("_cached_user", None)
         if "_user_pk" not in data:
             # We have a user but haven't stored its PK yet, so look it up and store it
-            data["_user_pk"] = data.pop("user").pk
-        else:
-            # We already have stored the PK, and we need to AVOID actually resolving the lazy user reference.
-            data.pop("user")
+            data["_user_pk"] = user.pk
         return data
```

Comment:
Wonder if this should be back-ported to 1.5.

Reply:
It might not be necessary, seeing as we don't have the recurring serialize/deserialize issue we have here with the database backend. The `_cached_user` only becomes a part of the payload when it is re-serialized from a deserialized payload a 2nd time.